admin_guide/out_of_resource_handling.adoc

```diff
@@ -43,7 +43,7 @@ An _eviction policy_ allows a node to fail one or more pods when the node is run
 Failing a pod allows the node to reclaim needed resources.
 
 An eviciton policy is a combination of an xref:out-of-resource-eviction-signals[eviction trigger signal] with a specific xref:out-of-resource-eviction-thresholds[eviction threshold value], that is set
-in the xref:out-of-resource-eviction-policy[node configuration file] or through the xref:out-of-resource-eviction-thresholds[command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. See the sections below for important information the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
+in the node configuration file or through the xref:out-of-resource-eviction-thresholds[command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. See the sections below for important information the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
```

Comment:
s/sections below/following sections
s/information the/information about the

Reply:
@kalexand-rh Which change are you asking about, related to Dedicated. The paragraph above or the PR in general? I did a search on `systemctl restart` in Dedicated (via the browser) and got no results. I can verify that all of the topics in this PR are not in Dedicated if you think there is a concern.

Reply:
In general. For example, the command in admin_guide/managing_projects.adoc is visible in the dedicated version of this built file, and I'm not sure if it should be.

Reply:
@kalexand-rh 
It appears that, of the files in the PR, only the Admin Guide -> Managing Projects and Setting Quotas topics are in Dedicated. Both appear to be properly ifdef'd to keep the restart commands out. Where are you seeing the commands in Managing Projects?

Reply:
@kalexand-rh I moved the endif to the end of the file. Page appears correct in Asciibinder preview